sqlmesh/core/model/kind.py

```diff
@@ -120,7 +120,7 @@ def is_symbolic(self) -> bool:
 
     @property
     def is_materialized(self) -> bool:
-        return not (self.is_symbolic or self.is_view)
+        return self.model_kind_name is not None and not (self.is_symbolic or self.is_view)
```

Comment:
why did this change?

Reply:
Enabling previews exposed a failure in tests when we attempted to restate standalone audit snapshots. The reason was the fact that `is_materialized` returned true for this kind of snapshots. `model_kind_name` is None in this case , but since the condition was a negation it returned true.